lms/djangoapps/courseware/views.py

```diff
@@ -821,6 +824,9 @@ def course_about(request, course_id):
 
     course_key = SlashSeparatedCourseKey.from_deprecated_string(course_id)
 
+    if hasattr(course_key, 'ccx'):
```

Comment:
Please add some comment to explain why you do not need to check if the user is logged in and/or enrolled.